src/components/calcite-flow-item/resources.ts

```diff
@@ -12,7 +12,8 @@ export const CSS = {
   menuButton: "menu-button",
   menu: "menu",
   menuOpen: "menu--open",
-  fabContainer: "fab-container"
+  fabContainer: "fab-container",
+  defaultSlotContainer: "default-slot-container"
```

Comment:
It doesn't seem like this class is being used.

Reply:
Yeah but Its better than a div without anything.

Reply:
Strong disagree. Let's not add anything that isn't being used.